[2024-12-06T09:31:25.920+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-06T09:31:25.928+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9868205040_2024-12-06T09-30-58-302821Z.update_schedule_flag scheduled__2024-12-06T09:30:58.302821+00:00 [queued]>
[2024-12-06T09:31:25.931+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9868205040_2024-12-06T09-30-58-302821Z.update_schedule_flag scheduled__2024-12-06T09:30:58.302821+00:00 [queued]>
[2024-12-06T09:31:25.931+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-06T09:31:25.936+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): update_schedule_flag> on 2024-12-06 09:30:58.302821+00:00
[2024-12-06T09:31:25.941+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=56798) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-06T09:31:25.942+0000] {standard_task_runner.py:72} INFO - Started process 56799 to run task
[2024-12-06T09:31:25.942+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9868205040_2024-12-06T09-30-58-302821Z', 'update_schedule_flag', 'scheduled__2024-12-06T09:30:58.302821+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/schedule_call.py', '--cfg-path', '/tmp/tmpv_dk34de']
[2024-12-06T09:31:25.943+0000] {standard_task_runner.py:105} INFO - Job 154: Subtask update_schedule_flag
[2024-12-06T09:31:25.965+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9868205040_2024-12-06T09-30-58-302821Z.update_schedule_flag scheduled__2024-12-06T09:30:58.302821+00:00 [running]> on host f4eb63bf5944
[2024-12-06T09:31:25.999+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9868205040_2024-12-06T09-30-58-302821Z' AIRFLOW_CTX_TASK_ID='update_schedule_flag' AIRFLOW_CTX_EXECUTION_DATE='2024-12-06T09:30:58.302821+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-06T09:30:58.302821+00:00'
[2024-12-06T09:31:26.000+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-06T09:31:26.004+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2024-12-06T09:31:26.009+0000] {sql.py:544} INFO - Running statement: UPDATE user_details SET status = 'called', modified_on_utc = now() at time zone 'utc' WHERE id = 33, parameters: None
[2024-12-06T09:31:26.010+0000] {sql.py:553} INFO - Rows affected: 1
[2024-12-06T09:31:26.011+0000] {logging_mixin.py:190} INFO - Marked record 33 as scheduled.
[2024-12-06T09:31:26.011+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-06T09:31:26.016+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-06T09:31:26.016+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9868205040_2024-12-06T09-30-58-302821Z, task_id=update_schedule_flag, run_id=scheduled__2024-12-06T09:30:58.302821+00:00, execution_date=20241206T093058, start_date=20241206T093125, end_date=20241206T093126
[2024-12-06T09:31:26.035+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-06T09:31:26.042+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-06T09:31:26.043+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
